src/package_io/registry.star

```diff
@@ -50,8 +50,10 @@ _DEFAULT_IMAGES = {
     OP_BATCHER: "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-batcher:develop",
     # Challenger
     OP_CHALLENGER: "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-challenger:develop",
-    # Supervisor
+    # op-supervisor
     OP_SUPERVISOR: "us-docker.pkg.dev/oplabs-tools-artifacts/images/op-supervisor:develop",
+    # kona-supervisor
+    KONA_SUPERVISOR: "ghcr.io/op-rs/kona/kona-supervisor:latest",
```

Comment:
You'll also gonna need to add a "registry id" - look above in this file and add a constant just like `OP_SUPERVISOR`